index.html

```diff
@@ -1,657 +1,1263 @@
+                                </div>
+                              </div>
+                            </div>
+                          </div>
                         </div>
-                    </a>
-                </div>
-                <div class="button resource-btn">
-                    <a href="" id="" class="btn-link" data-cmp-clickable="" data-cmp-data-layer="">
-                        <span class="btn-text">All Resources</span>
-                    </a>
+                      </div>
+                    </div>
+
+                    <div class="button aem-GridColumns aem-GridColumns--default--12">
+                      <a id="button-dbb21d03a5" class="cmp-button" href="/content/leaderimpact/lang/en2/resources.html">
+                        <span class="cmp-button__text">All Resources</span>
+                      </a>
+                    </div>
+                  </div>
                 </div>
+              </div>
             </div>
-        </section>
-        <section class="impact-story">
-            <div class="container">
-                <div class="stories-wrap">
-                    <div class="title">
-                        <div data-cmp-data-layer="" id="" class="title-wrap">
-                            <h2>Impact Stories</h2>
-                        </div>
+          </div>
+        </div>
+
+        <div class="container responsivegrid impact-story aem-GridColumns aem-GridColumns--default--12">
+          <div id="container-36618f7ece" class="cmp-container">
+            <div id="impact-container" class="aem-Grid aem-Grid--12 aem-Grid--default--12 ">
+              <div class="container responsivegrid stories-wrap aem-GridColumns aem-GridColumns--default--12">
+                <div id="container-112257af9a" class="cmp-container">
+                  <div class="aem-Grid aem-Grid--12 aem-Grid--default--12 ">
+                    <div class="title aem-GridColumns aem-GridColumns--default--12">
+                      <div id="title-63091b8133" class="cmp-title">
+                        <h2 class="cmp-title__text">Impact Stories</h2>
+                      </div>
                     </div>
-                    <div class="stories-item-wrap">
-                        <a href="#story1" class="story-item">
-                            <div class="img">
-                                <img srcset="./assets/story1.png" src="./assets/story1.png" loading="lazy" class="post-img" itemprop="contentUrl"  alt="" title="">
+
+                    <div class="container responsivegrid aem-GridColumns aem-GridColumns--default--12">
+                      <div id="container-9aefcfdaa3" class="cmp-container">
+                        <div id="stories-item-wrap" class="aem-Grid aem-Grid--12 aem-Grid--default--12 ">
+                          <div class="teaser story-item aem-GridColumns aem-GridColumns--default--12" data-val="#tabs-31c70c9208-item-b17065cfe5-tabpanel">

```

Comment:
These `data-val` properties will need to be customized with the javascript somehow, or we'll need to use IDs. If we add an ID, it would go on the line below this one, with the `cmp-teaser` class.